src/components/GitStars.js

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+import styled from "styled-components"
+import Emoji from "./Emoji"
+import Icon from "./Icon"
+import Link from "./Link"
+
+const Container = styled(Link)`
+  margin-right: -0.5rem;
```

Comment:
Generally I'm not a fan of negative margin - why is this needed here? If we want these components to be aligned further right, I suggest we reduct the padding of the card content.